agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/RuntimeConfigurator.java

```diff
@@ -27,14 +33,20 @@ public class RuntimeConfigurator {
   private final TelemetryClient telemetryClient;
   private final Supplier<AgentLogExporter> agentLogExporter;
   private volatile RuntimeConfiguration currentConfig;
+  @Nullable private final Consumer<List<TelemetryItem>> heartbeatTelemetryItemsConsumer;
+  @Nullable private final ProfilingInitializer profilingInitializer;
```

Comment:
looks like only `null` in tests, so instead I'd suggest passing no-op functions in tests and removing `@Nullable`